### PR TITLE
getFullSchedule - add current period

### PR DESCRIPTION
**Pull request by Tritium-VLK**
solves #35

### CODE REVIEW

File: contracts/ChildChainGaugeInjectorV2.sol [hunk 1]
```diff
@@ -355,10 +355,11 @@ KeeperCompatibleInterface
 /**
 * @notice Gets all current schedule information as a set of arrays
 */
-    function getFullSchedule() public view returns (address[] memory, uint256[] memory, uint8[] memory, uint56[] memory, uint56[] memory) {
+    function getFullSchedule() external view returns (address[] memory, uint256[] memory, uint8[] memory, uint56[] memory, uint56[] memory) {
```
Comment (petrovska-petro) [inline contracts/ChildChainGaugeInjectorV2.sol:358]: it's missing to return another item here; only returning 5 items, but expecting 6

File: contracts/ChildChainGaugeInjectorV2.sol [hunk 2]
```diff
@@ -355,10 +355,11 @@ KeeperCompatibleInterface
...
         address[] memory gauges = getActiveGaugeList();
         uint len = gauges.length;
         uint256[] memory amountsPerPeriod = new uint256[](len);
+        uint256[] memory currentPeriod = new unit256[](len);
```
Comment (petrovska-petro) [inline contracts/ChildChainGaugeInjectorV2.sol:362]: `periodNumber` in the struct `Target` is type `uint8` not `uint256`

File: contracts/ChildChainGaugeInjectorV2.sol (hunk 1 above)
Comment (Tritium-VLK) [reply contracts/ChildChainGaugeInjectorV2.sol]: thanks.  Test wasn't running and env vars still not working so hardhat also didn't work.  Will do a final check of everything, run tests, think about that once we get this all merged in.

File: contracts/ChildChainGaugeInjectorV2.sol (hunk 2 above)
Comment (petrovska-petro) [reply contracts/ChildChainGaugeInjectorV2.sol]: confirmed right type was added in 678cf27d7fafbde49ce691cf24943c61e85936c2 ✅

**Review by petrovska-petro — APPROVED** [review]
comments were addressed ✅

**Pull request merged**